docs/src/start/dylib.md

```diff
@@ -40,13 +56,15 @@ abi_stable = "0.11.1"
 sys-info = "0.9.1"
```

Comment:
The example uses `steel-core` from the workspace. However, maybe it would be best if we can point to a real version?

How long until we reach a possible stable major version? Or is there no minor/major distinction due to ABI issues?

Reply:
I agree, would be best to point to a real version.

I can cut a new release to publish to crates.io this weekend and we can use that version here if that's alright, I think that'll be 0.6.0 after I publish. I don't think I'm quite ready to cut a 1.0 just yet, but given growing usage I don't think that is as far off as I originally thought

Reply:
Ack. Makes sense.